src/main/java/com/box/sdk/BoxJSONObject.java

```diff
@@ -41,7 +42,7 @@ public BoxJSONObject() {
      * @param json the JSON string to decode.
      */
     public BoxJSONObject(String json) {
-        this(JsonObject.readFrom(json));
+        this(Json.parse(json).asObject());
```

Comment:
What's the benefit of this change?

Reply:
JsonObject.readFrom is deprecated and we should stop using it and replace usages with Json.parse(json).asObject().